docs/contributing/pull-requests/index.md

```diff
@@ -66,22 +66,7 @@ git checkout -b <pod>/<feature>/<issue-number> -t origin/master
 
 <bitwarden>
 
-### Naming convention
-
-To keep branches organized we adhere a naming convention for branches. The convention splits branch
-names into 2 components, each separated by a `/`.
-
-1. The developers pod abbreviation (e.g. `sm`)
-2. Jira issue tag (e.g. `SM-1234`)
-
-```bash
-# Some examples
-sm/SM-1234
-ps/PS-4321-optional-description
-```
-
-This naming convention allows us to easily identify the type of work being done on a branch. And
-will assist in identifying and tracking down stale branches.
+Our branching strategy is described in detail [here](branching.md).
```

Comment:
We could move this up into the previous section. Do you mind updating line 62 to follow the convention?

`git checkout -b <pod>/<jira-tag> -t origin/master`

Reply:
By "section", I think you meant set of <bitwarden> tags, correct?  If so, I've moved it.